br/pkg/lightning/restore/get_pre_info.go

```diff
@@ -54,6 +54,9 @@ import (
 	"golang.org/x/exp/maps"
 )
 
+// compressionRatio is the tikv/tiflash's compression ratio
+const compressionRatio = float64(1) / 3
```

Comment:
After asked tiflash member, the size of tikv is similar with tiflash for one replica